Vagrantfile

```diff
@@ -7,7 +7,7 @@ targets = {
   },
 
   "centos6.5" => {
-    "box" => "chef/centos-6.5"
+    "box" => "puphpet/centos65-x64"
```

Comment:
Looks like, [chef moved to centos-6.7](https://vagrantcloud.com/bento/boxes/centos-6.7).
They have old boxes available [here](https://github.com/chef/bento/blob/master/OLD-BOXES.md)

While we are at it, we should also update to their new namespace: `chef` -> `bento`